docs/graphite.md

```diff
@@ -142,7 +142,7 @@ See also:
 | removeBelowValue(seriesList, n) seriesList                     |              | Stable     |
 | removeBetweenPercentile                                        |              | No         |
 | removeEmptySeries                                              |              | No         |
-| roundFunction                                                  |              | No         |
+| round                                                          |              | Stable     |
```

Comment:
shouldn't this be roundFunction with 'round' as an alias?

Reply:
hmm the docs are confusing.
https://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.roundFunction implies `roundFunction()` works, but in my tests, it doesn't

Reply:
Yeah, I checked this as well. The docs seem to be incorrect and only `round` works

Reply:
This appears to be because the docs are generated and the function name is `roundFunction()` (https://github.com/graphite-project/graphite-web/blob/master/webapp/graphite/render/functions.py#L1674) while the key in the dict where those functions are looked up from is `round` (https://github.com/graphite-project/graphite-web/blob/master/webapp/graphite/render/functions.py#L5702)

I suggest that I'll make the docs match the behavior, and not the other way around. Otherwise we might break existing dashboards.